test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -627,7 +633,7 @@ int test_main(int, char* [])
     // Not yet fully tested for float and long double.
     // The difference algorithm can generate (additional) slivers
     // Many of the failures are self-intersection points.
-    BoostGeometryWriteExpectedFailures(12, 10, 17, 12);
+    BoostGeometryWriteExpectedFailures(19, 10, 17, 12);
```

Comment:
Side note: adding one case can indeed cause 7 extra reported issues (on number of geometries, area, validity, for a-b and b-a, s, etc)